### PR TITLE
docs: add "getting started" instructions to the readme

### PR DESCRIPTION
**Pull request by ramfox**
## Description

Add an install page with "Getting Started" instructions. Also, pull those instructions in to our generated docs.

This is a separate page from the README, since it gives us flexibility on how to import the content in other locations (rustdoc & iroh.computer/docs).

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Getting started
```
Comment (dignifiedquire) [inline INSTALL.md]: please just put this in the readme, a new file really doesn't make sense

File: src/lib.rs [hunk 2]
```diff
@@ -23,6 +23,7 @@
 //!
 //! [BLAKE3]: https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf
 //! [iroh]: https://docs.rs/iroh
+#![doc = include_str!("../INSTALL.md")]
```
Comment (dignifiedquire) [inline src/lib.rs]: lets include the readme here

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+The `iroh-blobs` protocol is meant to be use in conjunction with `iroh`. [Iroh](https://docs.rs/iroh/latest/iroh/index.html) is a networking library for making direct connections, these connections are what power the data transfers in `iroh-blobs`.
```
Comment (dignifiedquire) [inline INSTALL.md]: usually best to link to `docs.rs/iroh`, the redirects will take care of the rest

File: README.md [hunk 4]
```diff
@@ -21,6 +21,49 @@ This crate is usually used together with [iroh-net](https://crates.io/crates/iro
...
+
+```rust
+use iroh::{protocol::Router, Endpoint};
+use iroh_blobs::{net_protocol::Blobs, util::local_pool::LocalPool};
+
+#[tokio::main]
+async fn main() -> Result<(), std::fmt::Error> {
+    // create an iroh endpoint that includes the standard discovery mechanisms
+    // we've built at number0
+    let endpoint = Endpoint::builder().discovery_n0().bind().await.unwrap();
```
Comment (rklaehn) [inline README.md]: why unwrap?

**Review by rklaehn — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -56,10 +56,14 @@ async fn main() -> Result<(), std::fmt::Error> {
...
         .accept(iroh_blobs::ALPN, blobs.clone())
-        .spawn();
+        .spawn()
+        .await?;
 
     // do fun stuff with the blobs protocol!
     // make sure not to drop the local_pool before you are finished
+    router.shutdown().await?;
+    drop(local_pool);
+    drop(tags_client);
```
Comment (dignifiedquire) [inline README.md:66]: explicit drops? why?

**Pull request merged**